events/validations/list.js

```diff
@@ -1,7 +1,57 @@
-const { query } = require('express-validator/check');
+const { query, checkSchema } = require('express-validator/check');
+const moment = require('moment');
 const ValidationHelper = require('../../util/ValidationHelper');
 
 module.exports = [
-  query('query[dojo_id]', 'dojo_id must be a UUID').isUUID(),
+  checkSchema({
```

Comment:
page, pageSize and order will be used by any listing (applications as an example)
Dates rules are equivalent and will be reused -> externalize definition from mapping